addons/website/controllers/form.py

```diff
@@ -280,6 +280,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            _authenticate_message = False
```

Comment:
Can be named `authenticate_message`, I don't see why we should add `_` as prefix for that variable. 🤔